pkg/cmd/cli/config/smart_merge.go

```diff
@@ -0,0 +1,157 @@
+package config
```

Comment:
Almost a straight copy, I just moved `getUniqueName` into this file.

----- Original Message -----
> > @@ -0,0 +1,157 @@
> > +package config
> 
> Is this a straight copy of:
> https://github.com/openshift/origin/blob/master/pkg/cmd/experimental/login/smart_merge.go?
> Need to clean up the original?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/992/files#r24611203